.github/workflows/update.yaml

```diff
@@ -0,0 +1,47 @@
+name: Update
+
+# FIXME: Delete when the PR is ready
+on:
+  push:
+    branches: 
```

Comment:
I’d change this to a `paths:` key that includes this file, so when you tweak this file, voila.

Reply:
👍